Provide link to updated Chromium docs. (#604)
MicrosoftDocs/edge-developer: microsoft-edge/extensions/includes/deprecation-note.md

ms.author: msedgedevrel
ms.prod: microsoft-edge
ms.topic: include
ms.date: 04/17/2020
---
> [!NOTE]
> This documentation is for extensions for the [legacy version of Microsoft Edge][MicrosoftSupportEdgeLegacy]. For information on the new Microsoft Edge, see [Microsoft Edge (Chromium) Extensions][MicrosoftEdgeExtensionsChromiumIndex].

> [!IMPORTANT]

> > The new Microsoft Edge program is now open to accept chromium-based extensions for the new Microsoft Edge \(v77 and newer\). If you want to submit a new extension, visit [Publish to Microsoft Edge (Chromium) Addons Store][ExtensionsChromiumPublish] to learn about the submission process.  

Author: ```suggestion
> The new Microsoft Edge program is now open to accept chromium-based extensions for the new Microsoft Edge \(v77 or later\). If you want to submit a new extension, visit [Publish to Microsoft Edge (Chromium) Addons Store][ExtensionsChromiumPublish] to learn about the submission process.  
```
